var/spack/repos/builtin/packages/gmsh/package.py

```diff
@@ -45,66 +46,82 @@ class Gmsh(CMakePackage):
     variant('gmp',         default=True,  description='Enable GMP for Kbipack (advanced)')
     variant('cairo',       default=False, description='Enable Cairo to render fonts (experimental)')
     variant('compression', default=True,  description='Enables IO compression through zlib')
-    variant('med',         default=True,  description='Build with MED(HDF5)')
-    variant('mmg',         default=True,  description='Build with Mmg3d')
-    variant('netgen',      default=True,  description='Build with Netgen')
+    variant('med',         default=True,  description='Build with MED(HDF5) (system)')
+    variant('mmg',         default=True,  description='Build with Mmg3d (system)')
```

Comment:
It should generally be assumed that options are provided through spack; and `system` here doesn't obviously mean spack. Think of these descriptions as being output from `spack info` rather than from the perspective of a package configure script: they should reflect the user's point of view.
```suggestion
    variant('mmg',         default=True,  description='Build with Mmg3d')
```